libraries/AP_HAL_ChibiOS/hwdef/CubeRedPrimary/hwdef.dat

```diff
@@ -0,0 +1,276 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H757
+
+# MCU class and specific type
+MCU STM32H7xx STM32H757xx
+
+# USB setup
+USB_VENDOR 0x2DAE # ONLY FOR USE BY HEX! NOBODY ELSE
+USB_PRODUCT 0x1059
+USB_STRING_MANUFACTURER "CubePilot"
+USB_STRING_PRODUCT "CubeRed"
+
+define CORE_CM7
+define SMPS_PWR
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1069
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 128
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
+
+define HAL_LED_ON 0
+
+# Nnow some defines for logging and terrain data files.
+define HAL_BOARD_LOG_DIRECTORY "/APM/LOGS"
+define HAL_BOARD_TERRAIN_DIRECTORY "/APM/TERRAIN"
+
+# Enable FAT filesystem support (needs a microSD defined via SDMMC).
+define HAL_OS_FATFS_IO 1
+
+# Setup the IMU heater
+define HAL_HAVE_IMU_HEATER 1
+define HAL_IMU_TEMP_DEFAULT 45
+define HAL_IMUHEAT_P_DEFAULT 50
+define HAL_IMUHEAT_I_DEFAULT 0.07
+define HAL_IMU_TEMP_MARGIN_LOW_DEFAULT 5
+
+
+# supports upto 8MBits/s
+CANFD_SUPPORTED 8
+
+STM32_ST_USE_TIMER 12
+define CH_CFG_ST_RESOLUTION 16
+
+# use last 2 pages for flash storage
+# H757 has 16 pages of 128k each
+STORAGE_FLASH_PAGE 14
+define HAL_STORAGE_SIZE 32768
+
+# ADC setup
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PF11 BATT_CURRENT_SENS ADC1 SCALE(1)
+
+PB1 AUX_POWER ADC2 SCALE(1) # AUX Volt
+PF12 AUX_ADC2 ADC1 SCALE(1) # AUX Curr
+
+PF13 VDD_5V_SENS ADC2 SCALE(2)
+
+# And the analog input for airspeed (rarely used these days).
+PF3 PRESSURE_SENS ADC3 SCALE(2)
+PC3 FMU_SERVORAIL_VCC_SENS ADC3 SCALE(3)
```

Comment:
should be:
```
PC3 FMU_SERVORAIL_VCC_SENS ADC3 SCALE(3) ADC(xxx)
```
for some value of xxx, should also be on the ADC1 pins, and make user settable pins match the CubeOrange/Plus